### ISSUE TITLE
Does sencha docs use a private tool now?

### ISSUE THREAD

**Issue by ryan-nauman** [senchalabs/jsduck#661]
No more JSDuck there :(

**Comment by nene** [comment]
Indeed. I don't know for sure as I'm no more working at Sencha, but it looks like they have built a new UI for the docs, while JSDuck is probably still used to parse the information out of source code. Looking at the source, they seem to be stuck with ECMAScript 5, so JSDuck should work as an engine for them for now. At some point in the future they want to swap out the parser with something that supports ES6. But for like 4 years now there haven't been any interest from Sencha to do any improvements to JSDuck.

Sad that they haven't open-sourced it. I guess they've been headed towards more corporate and closed direction for a while now.
